packages/meditrak-app/app/home/HomeToolbarContainer.js

```diff
@@ -16,7 +16,7 @@ const HomeToolbar = ({ onNavigateToSurveysMenu, onNavigateToTupaiaWebsite, style
       textStyle={localStyles.buttonText}
     />
     <BlueButton
-      title="SURVEY FACILITIES"
+      title="SURVEY"
```

Comment:
@edmofro 2 months of refactors and now I'm finally able to make this change 😉

Reply:
Bahahahaha 😂